modeltranslation/tests/tests.py

```diff
@@ -463,6 +463,29 @@ def test_constructor(self):
         )
         self._test_constructor(keywords)
 
+    def test_update_or_create(self):
+        """
+        Test that update_or_create works as expected
+        """
+        # Create new object
+        obj = models.TestModel.objects.create(title_de='old de', title_en='old en')
+
+        # Update existing object
```

Comment:
Maybe add another test case when created is True?

Reply:
O.k., I added anotherone.